pricing-app/src/app/admin/orders/page.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+import { updateData } from "@/app/utils";
+import { FC, useEffect, useRef, useState } from "react";
+import OrderItem from "./orderItem";
+import { Options, Order } from "@/types/type";
+import moment from "moment";
+
+const OrderPage: FC = () => {
+  const [items, setItems] = useState<Order[]>([]);
```

Comment:
This is a bit old school, think about how to separated the view from the loading logic, something like: useOrder() 
https://www.freecodecamp.org/news/introduction-to-react-hooks/ ==> look at the custom hook

Reply:
I will improve it in another PR.